x/stakeibc/keeper/transfer.go

```diff
@@ -0,0 +1,102 @@
+package keeper
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	ibctypes "github.com/cosmos/ibc-go/v7/modules/apps/transfer/types"
+	clienttypes "github.com/cosmos/ibc-go/v7/modules/core/02-client/types"
+
+	"github.com/Stride-Labs/stride/v14/x/stakeibc/types"
+)
+
+// Transfers tokens from the community pool deposit ICA account to the host zone holding address for that pool
+func (k Keeper) IBCTransferCommunityPoolTokens(ctx sdk.Context, token sdk.Coin, communityPoolHostZone types.HostZone, shouldLiquidStake bool) error {
+
+	// The memo may contain autopilot commands to atomically liquid stake tokens when transfer succeeds
+	//  both transfer+liquid stake will succeed and stTokens will end in the stride side holding address, 
+	//  or neither will and the original base tokens will return to the foreign deposit ICA address
+	memoCommands := ""
+	autopilotCmd := "{ \"autopilot\": { \"receiver\": \"%s\",  \"stakeibc\": { \"action\": \"LiquidStake\" } } }"
+	if shouldLiquidStake {
```

Comment:
I think we can remove this parameter for the time being since we're only allowing LS's from the host denom

Reply:
So I have updated this in the recent push -- when staking we have the hostDenom, when redeeming we have the stHostDenom (or rather the ibc denom for it on the hostZone), but for ones triggered by the tx route, they can have any denom.  Thus the autopilot actions are more like (LiquidStake, RedeemStake, NoAction) in the current implementation

Reply:
makes sense!